.eslintrc.js

```diff
@@ -35,7 +35,12 @@ module.exports = {
       ],
       env: {
         jest: true,
-      }
+      },
+      globals: {
+        "$": "readonly",
+        "$$": "readonly",
+        "browser": "readonly",
```

Comment:
Can we mark these as `globals` only in the `tests/integration/` directory?

Reply:
@jrbenny35 Let me know if you need a hand with this. I think it should just be copying this one override config and creating a new one for `files: ['tests/integration/**/*.js']` with our fancy globals.

Reply:
Does it have to be named specially?

Reply:
@pdehaan ^

Reply:
Sorry, I'm not so good with words. I think the answer here is to remove these new globals from the "tests/**/*.js" glob here, and create a new entry with:

```js
    {
      files: [
        "tests/integration/**/*.js",
      ],
      globals: {
        "$": "readonly",
        "$$": "readonly",
        "browser": "readonly",
      },
    },
```